rh/schema.py

```diff
@@ -37,6 +88,18 @@ class RhForm(DjangoSessionFormMutation):
     class Meta:
         source = RhFormInfo
 
+    @classmethod
+    def perform_mutate(cls, form, info: ResolveInfo):
+        request = info.context
+        result = super().perform_mutate(form, info)
+        form_data = RhFormInfo.get_dict_from_request(request)
+        assert form_data is not None
+
+        full_address = form_data["address"] + ", " + form_data["borough"]
+        bbl, _, _ = lookup_bbl_and_bin_and_full_address(full_address)
+        request.session[RENT_STAB_INFO_SESSION_KEY] = get_rent_stab_info_for_bbl(bbl)
```

Comment:
now, we reset rent stab info on every mutate, addressing your comment earlier about having stale data present